tox.ini

```diff
@@ -12,6 +12,9 @@ deps =
     -r{toxinidir}/requirements/test.txt
 passenv =
     DISPLAY
+    BOKCHOY_HEADLESS
+    MOZ_HEADLESS
```

Comment:
What is the purpose of this variable?

Reply:
It's an alternative to the `-headless` command line argument to Firefox: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-the-firefox-addon-in-headless-mode .  It's not technically needed, I just added it to make sure people used to activating headless mode that way don't get confused by it not working when running tox.

Reply:
Ok, cool. Thanks for the clarification